server/.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
Let's also add .env to the gitignore to avoid accidentally pushing it

Reply:
Done